lib/ansible/plugins/callback/default.py

```diff
@@ -16,8 +16,25 @@
       - default_callback
     requirements:
       - set as stdout in configuration
+    options:
+      check_mode_markers:
+        name: Show markers when running on check mode
+        description:
+        - "Toggle to control displaying markers when running on check mode. The markers are 'DRY RUN'
+        at the beggining and ending of playbooks execution (when calling ansible-playbook --check)
+        and 'CHECK MODE' as a suffix at every task that is run in 'check' mode."
+        type: bool
+        default: yes
```

Comment:
version_added: '2.9'

Reply:
this was ok until a wroing push of me  (789620b to 6ed2867 3 hours ago), which is now again fixed (with the push  6ed2867 to 255f5c3)